internal/featuretests/v3/httpproxy.go

```diff
@@ -39,12 +39,13 @@ func headerContainsMatchCondition(name, value string, ignoreCase bool) contour_a
 	}
 }
 
-func headerNotContainsMatchCondition(name, value string, ignoreCase bool) contour_api_v1.MatchCondition {
+func headerNotContainsMatchCondition(name, value string, ignoreCase, treatMissingAsEmpty bool) contour_api_v1.MatchCondition {
 	return contour_api_v1.MatchCondition{
 		Header: &contour_api_v1.HeaderMatchCondition{
-			Name:        name,
-			NotContains: value,
-			IgnoreCase:  ignoreCase,
+			Name:                       name,
+			NotContains:                value,
+			IgnoreCase:                 ignoreCase,
+			TreatMissingHeadersAsEmpty: treatMissingAsEmpty,
```

Comment:
Nit: I think this variable can be renamed as `TreatMissingAsEmpty` to maintain uniformity with other fields.